nmostesting/TestHelper.py

```diff
@@ -231,12 +231,10 @@ def load_resolved_schema(spec_path, file_name=None, schema_obj=None, path_prefix
     else:
         base_uri_path = "file://" + base_path
 
-    loader = jsonref.JsonLoader(cache_results=False)

```

Comment:
Hmm, yeah, the jsonref 1.0.0 release includes https://github.com/gazpachoking/jsonref/pull/43... it's not clear from https://github.com/AMWA-TV/nmos-testing/pull/106 why we needed to set `cache_results=False`) and the jsonref 1.0.0 makes significant changes to the way `$ref`ed schemas are cached, so I can't immediately tell if this is going to cause us a problem.